index.html

```diff
@@ -105,7 +105,38 @@ <h2 class="proj-title bold font40-52">Tonic</h2>
             <li>css</li>
             <li>javaScript</li>
           </ul>
-          <button type="button" class="button 500">See Project</button>
+          <button id="btn1" type="button" class="button 500">See Project</button>
+        </div>
+      </div>
+      <div id="myModal1" class="modal">
+        <div class="modal-container">
+          <div class="modal-content">
+            <div class="modal-title">
+              <h1 class="proj-title bold font40-52">Tonic</h1>
+              <span class="xp1">&times;</span>
+            </div>
+            <div class="proj-subt d-flex modal-subt">
```

Comment:
- [ ] As requested in the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md#:~:text=you%20must%20use%20a%20JavaScript%20array%20to%20store%20all%20of%20the%20information%20for%20all%20projects.), kindly represent all projects as a JavaScript array of projects and dynamically render the project from the array.